erizo_controller/erizoController/erizoController.js

```diff
@@ -416,13 +416,13 @@ var listen = function () {
         //Gets 'sendDataStream' messages on the socket in order to write a message in a dataStream.
         socket.on('sendDataStream', function (msg) {
             if  (socket.room.streams[msg.id] === undefined){
-              log.warn('Trying to send Data from a non-initialized stream ', msg);
+              log.warn('message: Trying to send Data from a non-initialized stream, clientId: ' + socket.id + ' dataMsg:', msg);
```

Comment:
log.warn('message: Trying to send Data from a non-initialized stream, clientId: ' + socket.id + ', dataMsg:', msg);